target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -202,6 +203,7 @@ ramips_setup_macs()
 	xiaomi,mir3g)
 		lan_mac=$(mtd_get_mac_binary factory 0xe006)
 		;;
+	iodata,wn-dx1167r|\
 	xiaomi,mir3g-v2)
 		wan_mac=$(mtd_get_mac_binary factory 0xe006)
 		label_mac=$wan_mac
```

Comment:
This node should be moved to reensure correct sorting.

Reply:
fixed